src/cryptoadvance/specter/templates/includes/coin_selection.html

```diff
@@ -0,0 +1,201 @@
+<template id="coin-selection">
+    <span class="toggle_coinselection" style="cursor: pointer;">{{ _("Coin selection") }} {% if show_advanced_settings %}&#9660;{% else %}&#9654;{% endif %}</span>
```

Comment:
Not sure about intended behaviour, should coin selection only be selectable (down-pointing triangle aka &#9660) if advanced settings are set to True? In testing it looks like coin selection is independent from the advanced settings.